fe/src/main/java/org/apache/doris/qe/GlobalVariable.java

```diff
@@ -56,6 +56,11 @@
     @VariableMgr.VarAttr(name = "query_cache_size")
     private static long queryCacheSize = 1048576;
 
+    public static final String HEARTBEAT_FLAGS = "heartbeat_flags";
+    // the flags passed to Backends
+    @VariableMgr.VarAttr(name = "heartbeat_flags")
+    public static long heartbeatFlags = 0;
```

Comment:
I don't think it is a good idea to set this in one global variables. Maybe each single function has a corresponding variable. 
Because if there are some flags and you want to change one of them, then you should select all of them and change the result and set it back. This is error prone and not easy to use.
And may be some flag in the future can't be changed through variables.

Reply:
ok, I will use seperate variable.